code/game/turfs/open/space/space.dm

```diff
@@ -215,9 +215,9 @@ GLOBAL_VAR_INIT(starlight_color, COLOR_STARLIGHT)
 		if(RCD_CATWALK)
 			var/obj/structure/lattice/lattice = locate(/obj/structure/lattice, src)
 			if(lattice)
-				return list("mode" = RCD_CATWALK, "delay" = 0, "cost" = 1)
-			else
 				return list("mode" = RCD_CATWALK, "delay" = 0, "cost" = 2)
+			else
+				return list("mode" = RCD_CATWALK, "delay" = 0, "cost" = 4)
```

Comment:
why the changes they dont rly make sense

Reply:
talked to fikou about this over discord:

catwalks are built using iron rods on lattice.

lattice takes 1 iron rod, which is 2 matter, and the catwalk takes 1 iron rod, which is also 2 matter. totals to 4 matter for the full build.